packages/storybook-ui-components/stories/ExpandableText.stories.tsx

```diff
@@ -1,25 +1,37 @@
-import React from "react";
-import { storiesOf } from "@storybook/react";
+import React from 'react';
```

Comment:
What happened with the quotes here? I personally prefer single quotes, but it's a break with our standard linting.